frontend/packages/console-app/src/components/user-preferences/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default as UserPreference } from './UserPreferencePage';
```

Comment:
I would recommend to not rename components. Make it easier to find the right component / file.

```suggestion
export { default as UserPreferencePage } from './UserPreferencePage';
```

Reply:
missed that while renaming. Thanks!